src/indexer.cc

```diff
@@ -687,7 +690,7 @@ class IndexDataConsumer : public index::IndexDataConsumer {
   IndexDataConsumer(IndexParam &param) : param(param) {}
   void initialize(ASTContext &ctx) override { this->ctx = param.ctx = &ctx; }
 #if LLVM_VERSION_MAJOR < 10 // llvmorg-10-init-12036-g3b9715cb219
-# define handleDeclOccurrence handleDeclOccurence
+#define handleDeclOccurrence handleDeclOccurence
```

Comment:
Drop this difference...

Reply:
Done. Must be sneaked in by the formatter, didn't notice it. 😅